payu/laboratory.py

```diff
@@ -10,6 +10,17 @@
 
 from payu.fsops import mkdir_p, read_config
 
+LAB_INITIALIZE_ERROR = """
+The configured laboratory directory may not have write access. Edit/remove one
+(or more) of the following config.yaml options that determine the laboratory
+path:
+    - 'laboratory': Top-level directory for the model laboratory
+        Default: /scratch/${PROJECT}/${USER}/${MODEL}
+    - 'shortpath' Top-level directory for laboratory.
+        Default: /scratch/${PROJECT}
+    - 'project': The project to use for payu PBS jobs. Default: ${PROJECT}
```

Comment:
I think it makes sense to put `project` at the top, as the two other options use it, so it helps to have explained it already.

Reply:
Good point, I've added that in